apps/console/src/styles/app.css

```
@@ -532,3 +532,320 @@ input:focus, textarea:focus, select:focus {
+  align-items: center;
+}
+
+.wizard-content {
+  width: 100%;
+  max-width: 520px;
+  display: flex;
+  flex-direction: column;
+}
+
+.wizard-heading {
+  font-family: 'Lora', Georgia, serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Lora" (font-family-name-quotes)

(font-family-name-quotes)